chains/immutablezkevm/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://explorer.immutable.com/api-docs
+    family: blockscout
+    name: Immutable Explorer
+    url: https://explorer.immutable.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 13371
```

Comment:
ugh this is so annoying, this is the chain ID of our `test1` chain https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/sdk/src/consts/testChains.ts#L33

We'll need to either change test1 to use a different domain ID, or pick a different domainId here. The former is probably most reasonable lol

Having a conflict isn't an option bc of some stuff in the agents that require unique domain IDs, and probably some other ts places too